discovery-provider/src/api/v1/metrics.py

```diff
@@ -0,0 +1,93 @@
+import logging # pylint: disable=C0302
+from dateutil.parser import parse
+from flask import Flask, Blueprint
+from flask_restx import Resource, Namespace, fields, reqparse
+from src import api_helpers
+from src.api.v1.helpers import make_response, success_response, to_dict
+from .models.metrics import route_metric, app_name_metric, app_name
+from src.queries.get_route_metrics import get_route_metrics
+from src.queries.get_app_name_metrics import get_app_name_metrics
+from src.queries.get_app_names import get_app_names
+
+logger = logging.getLogger(__name__)
+
+ns = Namespace('metrics', description='Metrics related operations')
+
+route_metrics_response = make_response("metrics_reponse", ns, fields.List(fields.Nested(route_metric)))
+app_name_response = make_response("app_name_response", ns, fields.List(fields.Nested(app_name)))
+app_name_metrics_response = make_response("app_name_metrics_response", ns, fields.List(fields.Nested(app_name_metric)))
+
+metrics_route_parser = reqparse.RequestParser()
+metrics_route_parser.add_argument('path', required=True)
+metrics_route_parser.add_argument('query_string', required=False)
+metrics_route_parser.add_argument('start_time', required=True)
```

Comment:
would an optional end_time would be useful?

Reply:
Possibly, I'm not sure how this api is going to be consumed. 
I think the typical time-based pagination just has start_time and limit.